packages/generator/ni_measurement_plugin_sdk_generator/ni_measurement_plugin_client_generator/_constants.py

```diff
@@ -0,0 +1,36 @@
+"""Measurement plug-in client generator constants."""
+
+from google.protobuf.type_pb2 import Field
+from ni_measurement_plugin_sdk_service._internal.stubs.ni.protobuf.types.xydata_pb2 import (
+    DoubleXYData,
+)
+
+_V2_MEASUREMENT_SERVICE_INTERFACE = "ni.measurementlink.measurement.v2.MeasurementService"
+
+_INVALID_CHARS = "`~!@#$%^&*()-+={}[]\|:;',<>.?/ \n"
+
+_XY_DATA_IMPORT = "from ni_measurement_plugin_sdk_service._internal.stubs.ni.protobuf.types.xydata_pb2 import DoubleXYData"
+_PATH_IMPORT = "from pathlib import Path"
+
+_DEFAULT_IMPORT_TYPE = "In-Built"
+_CUSTOM_IMPORT_TYPE = "Custom"
+_IMPORT_MODULES: dict[str, str] = {}
```

Comment:
I get what you're trying to do using this `field` but I doubt that it's not the appropriate way. Should you consider creating this dictionary in the `template.py` and pass it by reference to the required methods?